ClientApp/src/components/NavMenu.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { Collapse, Container, Navbar, NavbarBrand, NavbarToggler, NavItem, NavLink } from 'reactstrap';
 import { Link } from 'react-router-dom';
-import './NavMenu.css';
+import './NavMenu.scss';
```

Comment:
NavMenu will be getting deleted so don't worry about this, may cause merge conflicts later

Reply:
no worries - was just meddling to test the contrast of our colours :)